chapters/request_size/request_size.rst

```diff
@@ -0,0 +1,87 @@
+=============================================
+ Understanding the Request Size parameter?
+=============================================
+
+In some use cases, you may wish to vary the number of ``Documents`` a single Request will receive. You can achieve this by adjusting the ``request_size`` parameter when setting the Flow. This guide covers the different configuration options Jina offers.
+
+.. contents:: Table of Contents
+    :depth: 2
+
+Request Size
+------------
+Jina defines the ``request_size`` as the parameter on client side. By adjusting the ``request_size`` in the Flow's API you could define the number of ``Documents`` contained in every ``Request``.
+
+Batch Size
+------------
+Batch size, commonly used in machine learning, often refers to the number of datapoints that we feed into the model in one iteration.
+
+In Jina ``batch_size`` is used by ``Drivers`` and ``Executors`` to guarantee that the ``Executor`` processes data in pieces of a specific size.
+
```

Comment:
you can refer to a glossary using :term:`Driver`, checkout glossary [here](https://docs.jina.ai/chapters/glossary/glossary.html). And [how](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/glossary.html) to use glossary.

Reply:
👍